front/lib/api/wrappers.ts

```diff
@@ -174,6 +183,38 @@ export function withPublicAPIAuthentication<T, U extends boolean>(
         });
       }
 
+      const bearerTokenRes = await getBearerToken(req);
+      if (bearerTokenRes.isErr()) {
+        return apiError(req, res, {
+          status_code: 401,
+          api_error: {
+            type: "not_authenticated",
+            message:
+              "The request does not have valid authentication credentials.",
+          },
+        });
+      }
+      const token = bearerTokenRes.value;
+      const authMethod = getAuthType(token);
+
+      // Authentification with Auth0 token.
+      // Straightforward since the token is attached to the user.
+      if (authMethod === "access_token") {
+        const auth = await Authenticator.fromAuth0Token({
+          token,
+          wId,
+        });
+        if (auth.isUser()) {
+          return handler(
+            req,
+            res,
+            auth,
+            null as U extends true ? Authenticator : null
+          );
+        }
```

Comment:
Still feels weird to not error here?

Reply:
Updated: if not user return error, else returns the handler. 👍🏻